src/framework/mocha/MochaRuntime.m

```diff
@@ -1322,8 +1322,6 @@ static bool MOBoxedObject_hasProperty(JSContextRef ctx, JSObjectRef objectJS, JS
 static JSValueRef MOBoxedObject_getProperty(JSContextRef ctx, JSObjectRef objectJS, JSStringRef propertyNameJS, JSValueRef *exception) {
     NSString *propertyName = (NSString *)CFBridgingRelease(JSStringCopyCFString(NULL, propertyNameJS));
 
-    Mocha *runtime = [Mocha runtimeWithContext:ctx];
```

Comment:
So `+[Mocha runtimeWithContext:]` is a relatively expensive call that we don't want to perform until definitely needed?

Reply:
It may be worth having a comment in the code to that effect.